src/config.py

```diff
@@ -45,6 +45,8 @@ class Configuration(NamedTuple):
     git_committer_username: str = "github-actions[bot]"
     git_committer_email: str = "github-actions[bot]@users.noreply.github.com"
     pull_request_title: str = "Update GitHub Action Versions"
+    pull_request_branch: str = "gh-actions-update"
+    pull_request_unique: bool = False
```

Comment:
I think we can remove this. If the user wants to use a specific branch then they can use `pull_request_branch` option. Otherwise defaults will be used.

Reply:
Done. Only `pull_request_branch` is used now.